enterprise-suite/es-grafana/jvm-exporter.json

```diff
@@ -3,19 +3,19 @@
     "graphName": "Heap",
     "promQL": [
       {
-        "expr": "sum by (kubernetes_pod_name)(jvm_heap_committed{ContextTags})",
+        "expr": "jvm_memory_bytes_committed{ContextTags,area=\"heap\"} or on (kubernetes_pod_name) jvm_heap_committed{ContextTags}",
         "legendFormat": "committed {{kubernetes_pod_name}}"
       },
       {
-        "expr": "sum by (kubernetes_pod_name)(jvm_heap_init{ContextTags})",
+        "expr": "jvm_memory_bytes_init{ContextTags,area=\"heap\"} or on (kubernetes_pod_name) jvm_heap_init{ContextTags}",
         "legendFormat": "init {{kubernetes_pod_name}}"
```

Comment:
Is there a reason to show "init"?

Reply:
This change was to merge jmx_exporter and cinnamon, and no effort was made to change the graphs otherwise.

I suppose someone tuning a JVM may be interested in init as a baseline, I dunno.